src/main/java/gregtech/api/unification/material/materials/OrganicChemistryMaterials.java

```diff
@@ -184,7 +184,7 @@ public static void register() {
         AllylChloride = new Material.Builder(1025, "allyl_chloride")
                 .fluid()
                 .color(0x87DEAA)
-                .components(Carbon, 2, Methane, 1, HydrochloricAcid, 1)
+                .components(Carbon, 3, Hydrogen, 5, Chlorine, 1)
```

Comment:
The components of this compound should be kept, as we use it specifically for autogenerated decomposition. The tooltip however should be changed to `C3H5Cl`.